src/plugins/workspace/public/application.tsx

```diff
@@ -25,7 +25,13 @@ export const renderCreatorApp = (
 ) => {
   ReactDOM.render(
     <OpenSearchDashboardsContextProvider services={services}>
-      <WorkspaceCreatorApp {...props} />
+      <Router>
+        <Switch>
+          <Route>
+            <WorkspaceCreatorApp {...props} />
+          </Route>
+        </Switch>
+      </Router>
```

Comment:
why do we need add router?

Reply:
I have the same concern that is the HashRouter here necessary?

Reply:
Yes, we need hashRouter to wrap switch and route.